content/src/main/content/jcr_root/apps/acs-commons/components/utilities/qr-code-generator/clientlibs/js.txt

```diff
@@ -0,0 +1,4 @@
+#base=js
+
+app.js
```

Comment:
see above; seems odd having these 2 in the same clientlib, unless im missing something?